crypto/s2n_ecdsa.c

```diff
@@ -32,8 +35,27 @@
 #include "crypto/s2n_openssl.h"
 #include "crypto/s2n_pkey.h"
 
+
 #define S2N_ECDSA_TYPE 0
 
+EC_KEY *s2n_unsafe_ecdsa_get_non_const(const struct s2n_ecdsa_key *ecdsa_key) {
+    if(ecdsa_key == NULL) {
+        return NULL;
```

Comment:
If you wanted to, you could make this a PTR_ENSURE_REF (error if ecdsa_key is NULL). Leaving it as-is is fine too.

My comments last time were about checking if ecdsa_key->ec_key was NULL, not checking if ecdsa_key was NULL. We use ecdsa_key in this method, but we don't use ecdsa_key->ec_key.

Reply:
PTR_ENSURE_REF +1